### ISSUE TITLE
(node:28688) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead

### ISSUE THREAD

**Issue by sarfarazansari** [gajus/write-file-webpack-plugin#68]
After upgrade to webpack 4 getting below warning

```
(node:28688) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
    at Object.apply (/Users/sarfarazansari/Projects/KISS-APP/ui/node_modules/write-file-webpack-plugin/dist/index.js:137:14)
    at webpack (/Users/sarfarazansari/Projects/KISS-APP/ui/node_modules/webpack/lib/webpack.js:37:12)
    at Object.<anonymous> (/Users/sarfarazansari/Projects/KISS-APP/ui/config/electron-dev.js:36:18)
```

**Comment by gajus** [comment]
@sarfarazansari Are you available to PR a change?

**Comment by sarfarazansari** [comment]
@gajus sorry to bother you.
But I guess I've fixed issue by [updating library 4.3.2](https://github.com/gajus/write-file-webpack-plugin/releases/tag/v4.3.2)
